e2e/cypress/test_module/cypress/features/dashboard-page/orders-filters/orders-filters.feature

```diff
@@ -0,0 +1,43 @@
+Feature: | Orders filters |
+
+  Background: Before each
+    Given I authenticate to the system under "ROLE_MANAGER" role
+    When I click on the Dashboard button
```

Comment:
We already have same step in `home-page-manager.steps.ts`, maybe also for admin. Please moe it to common steps and reuse

Reply:
Thank you. Reused